### PR TITLE
use block timestamp

### PR DESCRIPTION
**Pull request by alexcos20**
## Description

Use block.timestamp for DDO.created(when needed) and DDO.updated

### CODE REVIEW

**Comment by kremalicious** [comment]
It would be great to get this PR onto Rinkeby Aquarius, so we can test this together with https://github.com/oceanprotocol/market/pull/292

File: aquarius/app/util.py [hunk 1]
```diff
@@ -78,10 +78,11 @@ def reorder_services_list(services):
     return result
 
 
-def init_new_ddo(data):
+def init_new_ddo(data,timestamp):
     _record = copy.deepcopy(data)
-    _record['created'] = format_timestamp(data['created'])
+    _record['created'] = format_timestamp(datetime.fromtimestamp(timestamp).strftime('%Y-%m-%dT%H:%M:%S'))
```
Comment (ssallam) [inline aquarius/app/util.py:83]: could replace the time format string with this constant: `DATETIME_FORMAT_NO_Z`

File: aquarius/events/events_monitor.py [hunk 2]
```diff
@@ -420,9 +422,11 @@ def processUpdateDDO(self, event):
...
             return
 
-        _record = init_new_ddo(data)
-        _record['updated'] = get_timestamp()
-
+        _record = init_new_ddo(data,timestamp)
+        #make sure that we do not alter created flag
+        _record['created'] = asset['created']
+        # but we update 'updated'
+        _record['updated'] = format_timestamp(datetime.fromtimestamp(timestamp).strftime('%Y-%m-%dT%H:%M:%S'))
```
Comment (ssallam) [inline aquarius/events/events_monitor.py:429]: could replace the time format string with this constant: `DATETIME_FORMAT_NO_Z`

**Review by ssallam — APPROVED** [review]
lgtm

**Pull request merged**